index.html

```diff
@@ -269,8 +273,27 @@ <h2>Contact me</h2>
         </section>
       </main>
     </div>
-    <script src="app.js"></script>
   </div>
+  <!-- <div class="modal" id="myModal">
+    <div class="modalContent">
+      <div class="modalHeader">
+        <h2 class="modalTitle"></h2>
+        <span id="modalClose"></span>
+      </div>
+      <div class="modalInfo"></div>
+      <div class="modalBody">
+        <p class="modalDescription"></p>
+        <div class="modalMiddle">
+          <div class="modalTechnology"></div>
+          <div class="modalFooter">
+            <button class="source"><span class="sourceImg"><img src="" alt="image1" class="img1"></span></button>
+            <button class="life"><span class="lifeImg"><img src="" alt="image2" class="img2"></span></button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div> -->
```

Comment:
- [ ] Remove all the commented code from your project since it makes code not readable.

Reply:
:heavy_check_mark: Well Done